emukit/examples/preferential_batch_bayesian_optimization/pbbo/gp_models.py

```diff
@@ -0,0 +1,468 @@
+import numpy as np
+
+import matplotlib
+matplotlib.use('Agg')
```

Comment:
Is it fixing the same thing that installation is supposed to fix on OSX? If so, let's just leave it there, so that code itself doesn't force anything. If people found an alternative way to fix the issue, or didn't have the issue in the first place, hardcoding the backend may cause more harm than good.

Reply:
We actually don't need matplotlib at all so we can also remove this. Thanks for noticing!